packages/single-line/src/styles.ts

```diff
@@ -6,7 +6,15 @@ export const validationRow = css({
   justifyContent: 'space-between',
   fontSize: tokens.fontSizeM,
   marginTop: tokens.spacingXs,
-  color: tokens.gray700,
+  color: tokens.gray500,
+});
+
+export const counterRow = css({
```

Comment:
Nit: this is also identical but one attribute. Can we avoid this duplication somehow?

e.g. 
```javascript
export const validationRow = (align) => css({
  display: 'flex',
  justifyContent: align === 'value'? 'space-between': 'flex-end',
  fontSize: tokens.fontSizeM,
  marginTop: tokens.spacingXs,
  color: tokens.gray500,
});
```

Reply:
Actually we could get around needing the two styles/align flag by doing
`<span>{constraints.isDefaultConstraint || !enabled ? '' : `Max. ${constraints.max} characters`}</span>` in CharValidation. Any preference?